### DISCUSSION TITLE
Settings Menu slow to open

### DISCUSSION THREAD

**Discussion by Karl-Franz** [apohl79/audiogridder#1265] [Help]
The more presets I create, the longer it takes for the menu to pop up after I click the Settings Menu button on the AudioGridder Plugin window. I have 165 presets right now and it takes 5 seconds for the menu to appear.

**Comment by Karl-Franz** [comment]
Bump

**Comment by apohl79** [comment]
Created a bug issue for this, thx for reporting!
